app/models/miq_expression.rb

```diff
@@ -1779,15 +1779,9 @@ def self.atom_error(field, operator, value)
                                                                                     :error_message => err.message}
       end
       return false
-    when :ruby
-      return _("Ruby Script must not be blank") if value.blank?
-      return false
     else
-      return false
+      return _("Not valid field")
```

Comment:
I'm not sure if this is what we want here. There may be more data types coming from the column info than what is listed in the case statement. The original code may have just returned false (no error) for data types it doesn't know about. It may be safer to leave it that way.